python/lsst/validate/drp/calcnonsrd/model_phot_rep.py

```diff
@@ -71,14 +73,17 @@ def measure_model_phot_rep(metrics, filterName, matchedDataset, snr_bins=None):
         for idx_bins, ((snr_one_min, snr_one_max), (snr_two_min, snr_two_max)) in enumerate(snr_bins):
             bin_base = 1 + 2*idx_bins
             for source in name_sources:
-                reduceSources(matchedDataset, matchedDataset._matchedCatalog, extended=source == 'Gal',
-                              nameFluxKey=name_flux, goodSnr=snr_one_min, goodSnrMax=snr_one_max,
-                              safeSnr=snr_two_min, safeSnrMax=snr_two_max)
+                matches = filterSources(
+                    matchedDataset._matchedCatalog, keys=keys, extended=source == 'Gal',
```

Comment:
For clarity, maybe extended=source == 'Gal' should be rewritten as extended=(source == 'Gal')